fire/SFParamsMod.F90

```diff
@@ -28,6 +28,7 @@ module SFParamsMod
    real(r8),protected, public :: SF_val_max_durat
    real(r8),protected, public :: SF_val_durat_slope
    real(r8),protected, public :: SF_val_drying_ratio
+   real(r8),protected, public :: SF_val_fire_threshold    ! threshold for fires that spread or go out. KW/m (Pyne 1986)
```

Comment:
```suggestion
   real(r8),protected, public :: SF_val_fire_threshold    ! threshold for fires that spread or go out. kW/m (Pyne 1996)
```

Reply:
The default of 50 kW/m comes from SPITFIRE implementation in Thonicke et al 2010 per Pyne 1996, but maybe that does not need to go into the comments? More than likely we will not use that 50 kW/m value for all cases, so this reference may not make sense to keep.